applications/omnipresence/src/omnip_subscriptions.erl

```diff
@@ -407,6 +407,7 @@ subscribe_to_record(JObj) ->
                         ,subscription_id=wh_json:get_value(<<"Subscription-ID">>, JObj)
                         ,proxy_route= wh_json:get_value(<<"Proxy-Route">>, JObj)
                         ,version=Version
+                        ,user_agent=wh_json:get_value(<<"User-Agent">>, JObj)
```

Comment:
Ok. And what about `get_value` vs. `get_binary_value`? (here `user_agent :: api_binary()`)